### ISSUE TITLE
crash on free(buffers[i]);

### ISSUE THREAD

**Issue by FANWENBIN** [syedhali/EZAudio#186]
```
+ (void)freeFloatBuffers:(float **)buffers numberOfChannels:(UInt32)channels
{
    for (int i = 0; i < channels; i++)
    {
        free(buffers[i]);   // crash on this line,maybe you should check buffers == NULL?
    }
    free(buffers);
}
```

crash when buffers = NULL

**Comment by syedhali** [comment]
There's a PR for this #180 . Going to merge it in now and bump the version to 1.0.1

**Comment by syedhali** [comment]
Fixed now, check out the 1.0.1 release
